### PR TITLE
Added: (boilerplate) more layouts and components so new users have more...

### PR DESCRIPTION
**Pull request by MoOx**
…example to show list of entries.

screen shot 2016-04-15 at 07 21 26

No sure why, for the first page (server side rendered), I got a react warning:

```
Warning: Failed propType: Invalid prop `date` of type `date` supplied to `PagePreview`, expected `string`. Check the render method of `PagesList`.
```

No idea why I get a "date" for the date field. gray-matter is not converting date (I checked the code), and client side, clearly have string, so no fucking idea.

### CODE REVIEW

File: boilerplate/web_modules/layouts/Homepage/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    collection: PropTypes.array.isRequired,
+  }
+
+  render() {
+    const latestPosts = enhanceCollection(this.context.collection, {
+      filter: { layout: "Post" },
+      sort: "date",
+      reverse: true,
+    })
+    .slice(0, numberOfLatestPosts)
```
Comment (thangngoc89) [inline boilerplate/web_modules/layouts/Homepage/index.js]: enhanceCollection has a limit option for this job.

Maybe we can extend it to support pagination

Comment (MoOx) [inline boilerplate/web_modules/layouts/Homepage/index.js]: Pagination will be another thing, later :)
Will use limit :)

**Comment by thangngoc89** [comment]
I ran into this before. Can't figure out what's going on.

**Comment by thangngoc89** [comment]
Not sure this is a bug or a feature. This came from gray-matter's dep js-yaml. It will parse date automatically. See https://github.com/dplepage/js-yaml/blob/5f058c3a78fd1a1bd36fd2cdc2c3a8c45ab53fd1/lib/js-yaml/type/timestamp.js#L79

The data which is sent to the client was JSON.stringify so it's a string

**Comment by MoOx** [comment]
I tried to assume it's a date, but somewhere else (client side), dates are string, so not consistent with server cache, which have real dates. We should normalize this. Opening an issue.

**Comment by thangngoc89** [comment]
Could we merge this and deal with the auto parsing date later ?

**Pull request merged**